Bindings/Java/tests/TestEditMarkers.java

```diff
@@ -0,0 +1,64 @@
+import org.opensim.modeling.*;
+import java.io.IOException;
+import static java.nio.file.StandardOpenOption.*;
+import java.nio.file.*;
+
+class TestEditMarkers {
+  public static void main(String[] args) {
+    try {
+        Model model = new Model("gait10dof18musc_subject01.osim");
+        OpenSimContext context=null;
+        long t1  = System.currentTimeMillis();
+        context = new OpenSimContext(model.initSystem(), model);
+        MarkerSet markerset = model.getMarkerSet();
+        // The following code exercises the Marker->Add New  functionality
+        Vec3 offset = new Vec3(0.11, 0.22, 0.33);
+        Body body = model.getBodySet().get(0);
+        String newMarkerName = "newMarker";
+        // This exercises Marker -> Add New
+        Marker newMarker = new Marker();
+        newMarker.setName(newMarkerName);
+        newMarker.set_location(offset);
+        newMarker.setParentFrame(body);
+        context.cacheModelAndState();
+        markerset.adoptAndAppend(newMarker);
+        try {
+            context.restoreStateFromCachedModel();
+        } catch (IOException ex) {
+            System.exit(1);
+        }
+        // This exercises Marker -> delete
+        context.cacheModelAndState();
+        Marker findMarker = Marker.safeDownCast(model.getComponent("newMarker"));
```

Comment:
Why do use the generic `getComponent` facility when you have the markerset in hand? Seems unnecessarily complicated.